Update reference docs (#1357)
tinacms/tinacms.org: content/docs/tina-cloud/data-layer.md

"start": "yarn tinacms server:start -c \"next start\" --experimentalData"
```

> To opt into the frontend features of the datalayer you can set a flag in the `cmsCallback`.

Reviewer: ```suggestion
To opt-in to the frontend features of the datalayer you must set the `experimentalData` flag in the `cmsCallback` to true.
```
